examples/chef/chef.py

```diff
@@ -85,21 +92,107 @@ def load_config() -> None:
 
 def check_python_version() -> None:
     if sys.version_info[0] < 3:
-        print('Must use Python 3. Current version is ' +
-              str(sys.version_info[0]))
+        flush_print('Must use Python 3. Current version is ' +
+                    str(sys.version_info[0]))
         exit(1)
 
 
+def check_zap(master: bool = False) -> str:
+    """Produces hash of ZAP submodule.
+
+    Args:
+      master: Check branch master instead of current branch.
```

Comment:
`zap_check_master=True` is no longer used in any call to `generate_device_manifest()`.

Reply:
Although it was not called, I have removed the functionality to check master.

Reply:
For posterity: noting that leaving this in may have been useful later on. It could have been called in generate_zzz to make sure the user was using the latest zap version (would need an extra call to `git remote` to ensure the fetch works reliably and as intended). This would be useful to inform the user they have an outdated ZAP version right away, instead of on PR or merge master. However, as this would require touching a user's git configuration to work reliably I did not proceed.